tests/conftest.py

```diff
@@ -146,7 +146,9 @@ def evm_script_executor(owner, easy_track, calls_script, EVMScriptExecutor):
 
 
 @pytest.fixture(scope="module")
-def reward_programs_registry(owner, voting, evm_script_executor_stub, RewardProgramsRegistry):
+def reward_programs_registry(
```

Comment:
Could you please set max line width to 100 and/or do not re-format the existing code? 100 chars line limits seems to be more consistent with the rest codebase (at least with the limits parts).